src/containers/trends/TrendsContainer.js

```diff
@@ -342,17 +341,24 @@ export class TrendsContainer extends React.Component {
   }
 }
 
-export function mapStateToProps(state) {
+export function mapStateToProps(state, ownProps) {
+  const userId = _.get(ownProps, 'currentPatientInViewId');
   return {
-    viz: state.viz,
+    trendsState: _.get(state, ['viz', 'trends', userId], {}),
   };
 }
 
-export function mapDispatchToProps(dispatch) {
+export function mapDispatchToProps(dispatch, ownProps) {
   return bindActionCreators({
-    focusTrendsCbgSlice: actions.focusTrendsCbgSlice,
-    markTrendsViewed: actions.markTrendsViewed,
-    unfocusTrendsCbgSlice: actions.unfocusTrendsCbgSlice,
+    focusTrendsCbgSlice: actions.focusTrendsCbgSlice.bind(
+      null, ownProps.currentPatientInViewId
```

Comment:
Yeah I suppose but we'd want to be consistent about it, no? Doesn't look like we can enforce a standard with [eslint-plugin-lodash](https://github.com/wix/eslint-plugin-lodash). Shall I try to change all instances (I know I do it elsewhere, probably in CBGSlice.js) and we'll all try to keep an eye out in reviews in future or should I leave them all?